app/models/work.rb

```diff
@@ -0,0 +1,21 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  validates :title, presence: true
+  validates :category, presence: true
+  validates_uniqueness_of :title, scope: :category
+
+  def self.sort_by_vote_order
+    work = Work.all
+    return work.sort_by{|work| work.votes.count}.reverse
```

Comment:
Just noting that it's more efficient to do this with `.order` and other methods to have Postgres do the filtering and sorting.  Postgres is much better at it.